setup.py

```diff
@@ -363,8 +363,8 @@ def get_sphinx_theme_version() -> str:
     'pymssql~=2.1,>=2.1.5',
 ]
 mysql = [
-    'mysql-connector-python>=8.0.11, <=8.0.22',
-    'mysqlclient>=1.3.6,<1.4',
+    'mysql-connector-python>=8.0.11, <=8.0.18',
```

Comment:
why do you want to use older version of `mysql-connector-python`?
see https://github.com/apache/airflow/pull/13370

Reply:
this PR is older then that